configure.py

```diff
@@ -1779,16 +1789,16 @@ def configure_abseil(build_dir, mode):
         build dist-tools-deb: build-submodule-deb tools/java/build/scylla-tools-package.tar.gz
           dir = tools/java
           artifact = $builddir/scylla-tools-package.tar.gz
-        build dist-tools-tar: phony {' '.join(['$builddir/{mode}/dist/tar/scylla-tools-package.tar.gz'.format(mode=mode) for mode in build_modes])}
+        build dist-tools-tar: phony $builddir/{mode}/dist/tar/scylla-tools-package.tar.gz
         build dist-tools: phony dist-tools-tar dist-tools-rpm dist-tools-deb
 
         rule compat-python3-reloc
-          command = mkdir -p $builddir/release && ln -f $dir/$artifact $builddir/release/
+          command = mkdir -p $builddir/release && ln -f $dir/$artifact $builddir/{mode}/
```

Comment:
Why is this change needed? Please note that you also need to fix the `mkdir` path not to be `release` if we do this.